frontend-react/src/components/Admin/OrgSenderTable.test.tsx

```diff
@@ -165,7 +165,9 @@ describe("OrgReceiverTable", () => {
     afterEach(() => settingsServer.resetHandlers());
     afterAll(() => settingsServer.close());
     beforeEach(() => {
-        renderWithRouter(<OrgReceiverTable orgname={"test"} key={"test"} />);
```

Comment:
Oh, I just noticed. This is the WRONG test. It should be the **_Sender_** table.

Reply:
Fixed.